backend/covid_resource_matcher/settings.py

```diff
@@ -113,3 +113,6 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = "/static/"
+
+# Redirect to home URL after login (Default redirects to /accounts/profile/)
+LOGIN_REDIRECT_URL = '/covid/profile'
```

Comment:
Would you mind fixing the trailing newlines in a few of this files? Otherwise LGTM

Reply:
I thiiiiiink I got all that cleared up. I've always defaulted to a trailing `\n` for the purposes of better, easier `cat`ing, but this should be good?